lib/components/table.vue

```diff
@@ -136,12 +135,24 @@
                 default() {
                     if (this && this.itemsProvider) {
                         // Deprecate itemsProvider
-                        warn("b-table: prop 'items-provider' has been deprecated. Pass a function to 'items' instead");
+                        warn('b-table: prop \'items-provider\' has been deprecated. Pass a function to \'items\' instead');
                         return this.itemsProvider;
                     }
                     return [];
                 }
             },
+            sortBy: {
+                type: String,
+                default: null
+            },
+            sortDesc: {
+                type: Boolean,
+                default: true
+            },
```

Comment:
You might get prop mutation errors on `sortBy` and `sortDesc` due to the listeners on head click.

You might need local copies of these in data, and then use the new `.sync` modifiers on the props, and emit `updated:sort-by` and `update:sort-desc` events to update the users copy of the props.